packages/manager-components/src/components/table/table.component.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { PropsWithChildren } from 'react';
+import style from './table.module.scss';
+
+export type TableProps = PropsWithChildren<{
+  fullWidth?: boolean;
+  cellSpacing?: number;
+}>;
+
+export const Table: React.FC<TableProps> = ({
+  children,
+  fullWidth = false,
+  cellSpacing = 0,
+}) => (
+  <table
+    className={`${style.table} ${fullWidth ? 'w-full' : ''}`}
+    cellSpacing={cellSpacing}
+  >
+    {children}
+  </table>
```

Comment:
**question:** Instead of accepting rows as the children, can we build the rows (`tr` -> `td`) in the Table components itself, that reduces the refactoring to ODS datagrid.

Reply:
Sure we can do this, but in 'Tanstack query', we can use 'cell' attribute in 'columns' to display custom component.

It can be restrictive to not let developer customize headers for sorting , making columns selectable, or adding columns ordering

At this moment, we are not mature inside our projects to define the best way to display datagrid, that`s why I only put a simple table with css,  developer can customize as project needed